packages/cashier/src/Containers/deposit.jsx

```diff
@@ -79,7 +80,7 @@ const Deposit = ({
     if (is_virtual) {
         return <Virtual />;
     }
-    if (is_deposit) {
+    if (is_deposit || is_eu) {
```

Comment:
If this is also flickering, it might be good to use `is_landing_company_loaded` here too.

Reply:
the <Loading /> happens before this, so I believe only that one is enough